web/src/frontend/src/nav-soknad/styles/skjema.less

```diff
@@ -23,4 +23,9 @@
 		padding-left: 0;
 		padding-right: 0;
 	}
+	.hoyretekst {
```

Comment:
midlertidig hack for å få prosenttegnet etter inputfeltet men på samme linje

Reply:
Må nok endres ja, men får høre om vi får det inn i frontend-moduler